todotxt.go

```diff
@@ -107,28 +108,29 @@ func (tasklist *TaskList) RemoveTask(task Task) error {
 	return nil
 }
 
-// LoadFromFile loads a TaskList from *os.File.
+// LoadFrom loads a TaskList from an io.Reader
 //
-// Using *os.File instead of a filename allows to also use os.Stdin.
+// os.File and os.Stdin are interfaces that include io.Reader, which is also bufio.Scanner's input type
 //
-// Note: This will clear the current TaskList and overwrite it's contents with whatever is in *os.File.
-func (tasklist *TaskList) LoadFromFile(file *os.File) error {
+// Note: This will clear the current TaskList and replace its contents with the reader's output.
+func (tasklist *TaskList) LoadFrom(tasks io.Reader) error {
 	*tasklist = []Task{} // Empty task list
 
 	taskID := 1
-	scanner := bufio.NewScanner(file)
+	scanner := bufio.NewScanner(tasks)
+
+    // scan in each line
 	for scanner.Scan() {
 		text := strings.Trim(scanner.Text(), whitespaces) // Read line
-
-		// Ignore blank or comment lines
-		if isEmpty(text) || (IgnoreComments && strings.HasPrefix(text, "#")) {
-			continue
-		}
-
 		task, err := ParseTask(text)
+
 		if err != nil {
-			return err
-		}
+            return err
```

Comment:
Space issue. Ditto.

Reply:
Looks like my editor formatting is acting up. I'll just `gofmt` it manually.